src/components/Modals/Send/views/Details.tsx

```diff
@@ -81,10 +81,13 @@ export const Details = () => {
   } = useWallet()
 
   useEffect(() => {
+    // This component initially mounts without an accountId, because of how <AccountDropdown /> works
+    if (!accountId) return
     // Initial setting of cryptoAmount in case of a QR-code set amount
-    if (!cryptoAmount) handleInputChange(cryptoAmount ?? '0')
+    handleInputChange(cryptoAmount ?? '0')
+    trigger(SendFormFields.CryptoAmount)
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [handleInputChange])
+  }, [accountId])
```

Comment:
This will probably break things again, test thoroughly before opening.

The most likely to be correct heuristics here are a `usePrevious` on accountId, and only run if `(!previousAccountId && accountId)`

Reply:
b5846fd540030dea81e58eaf420ccd648d426767